torch/nn/modules/pooling.py

```diff
@@ -70,10 +75,6 @@ def forward(self, input):
                             self.padding, self.dilation, self.ceil_mode,
                             self.return_indices)
 
-    def extra_repr(self):
```

Comment:
why do we remove this?

Reply:
This is a duplicate from the parent -- it is inherited